evaluator/evaluate.go

```diff
@@ -364,7 +364,8 @@ func evaluateAndStore(tx *gorm.DB, system *models.SystemPlatform,
 
 	// Send instant notification with new advisories
 	if enableInstantNotifications {
-		err = publishNewAdvisoriesNotification(tx, system.InventoryID, accountName, system.RhAccountID, newSystemAdvisories)
+		err = publishNewAdvisoriesNotification(tx, system.InventoryID, accountName, orgID,
+			system.RhAccountID, newSystemAdvisories)
```

Comment:
And same here :) what about just  `publishNewAdvisoriesNotification(tx, system.InventoryID, event, newSystemAdvisories)`.